plugins/j2store/report_products/report_products/tmpl/default.php

```diff
@@ -179,7 +179,7 @@
                             <tbody>
                             <tr>
                                 <td><?php echo $product->orderitem_name; ?>
-                                    <br>
+                                    <br/>
```

Comment:
Please revert this change. Joomla uses html where the self closing slash is not used. This is xhtml

Reply:
I only touched the language string as it was hijacking a core j2store language string (J2STORE_FILTER_SEARCH) but I must not have synced the branch so reverting now.